lib/syspro/business_objects/portor.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'syspro/business_objects/parsers/portor_parser'
+require 'erb'
+
+module Syspro
+  module BusinessObjects
+    class PorTor < ApiResource
+      include Syspro::ApiOperations::Transaction
+      include Syspro::BusinessObjects::Parsers
+
+      attr_accessor :transaction_date, :ignore_warnings, :non_stocked_wh_to_use, :grn_matching_action,
+        :allow_blank_supplier, :apply_if_entire_document_valid, :validate_only,
+        :manual_serial_transfers_allowed, :ignore_analysis
+
+      def call(user_id)
+        raise NotImplementedError.new("PORTOR not implemented yet.")
+        #xml_parameters = params_template.result(binding)
```

Comment:
Can we remove this comment block?

Reply:
Need to implement this later, so I would rather leave it.